src/concrete/ml/pytest/torch_models.py

```diff
@@ -1106,7 +1092,9 @@ def forward(self, x):
 
         # Add an additional operator that requires a TLU in order to force this circuit to
         # handle a PBS without actually changing the results
-        torch_sum = torch_sum + torch_sum % 2 - torch_sum % 2
+        if self.with_pbs:
+            torch_sum = torch_sum + torch_sum % 2 - torch_sum % 2  # pragma: no cover
```

Comment:
can you add a comment to explain why this part is not tested, pls?

Reply:
will do in a following PR yes, it's tested in the weekly